### ISSUE TITLE
Improvements to Tooltip component for Input

### ISSUE THREAD

**Issue by alexandramfedyk** [telusdigital/tds#481]
Location
TDS component

Expected Behaviour
tooltip should be announced as soon as toggle is triggered by text to speech engine (screen reader)

Actual Behaviour
screen reader announces button state first. Message is announced after button state is complete.

How to Reproduce?
tab to component with screen reader
activate tooltip toggle
wait for toggle announcement to finish
tooltip content is announced
Which version of TDS does this effect?
TDS v0.29.0
